hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/committers.md

```diff
@@ -356,19 +351,20 @@ task commit.
 
 However, it has extra requirements of the filesystem
 
-1. [Obsolete] It requires a consistent object store.
+1. The object store must be consistent.
 1. The S3A client must be configured to recognize interactions
-with the magic directories and treat them specially.
+with the magic directories and treat them as a special case.
 
-Now that Amazon S3 is consistent, the magic committer is enabled by default.
+Now that [Amazon S3 is consistent](https://aws.amazon.com/s3/consistency/),
+the magic committer is enabled by default.
```

Comment:
File committer is still the default committer, right? So what does magic committer being "enabled by default" here mean?

Reply:
Yes, I believe that's correct.

I believe what we're saying here is that S3A's "magic path rewriting" where it only stages the writes to `__magic/` directories is now enabled by default.

I will update this to be clearer, something like:

```suggestion
the magic directory path rewriting is enabled by default.
```